packages/@guardian/source-foundations/src/typography/typography.stories.mdx

```diff
@@ -60,49 +73,39 @@ typography API assigns font size in rems.
 ```css
 font-family: 'GH Guardian Headline, Guardian Egyptian Web, Georgia, serif';
 ```
-
--   `headline.xxxsmall()` -> 17px
--   `headline.xxsmall()` -> 20px
--   `headline.xsmall()` -> 24px
--   `headline.small()` -> 28px
--   `headline.medium()` -> 34px
--   `headline.large()` -> 42px
--   `headline.xlarge()` -> 50px
+<Story name="headline">
+	<FontStylesRenderer fontName="headline" fontStyles={headlineObjectStyles} />
+</Story>
 
 ### Body
 
 ```css
 font-family: 'GuardianTextEgyptian, Guardian Text Egyptian Web, Georgia, serif';
 ```
 
--   `body.xsmall()` -> 14px
--   `body.small()` -> 15px
--   `body.medium()` -> 17px
+<Story name="body">
+	<FontStylesRenderer fontName="body" fontStyles={bodyObjectStyles} />
+</Story>
 
 ### Text sans
 
 ```css
 font-family: 'GuardianTextSans, Guardian Text Sans Web, Helvetica Neue, Helvetica, Arial, Lucida Grande, sans-serif';
 ```
 
--   `textSans.xxsmall()` -> 12px
--   `textSans.xsmall()` -> 14px
--   `textSans.small()` -> 15px
--   `textSans.medium()` -> 17px
--   `textSans.large()` -> 20px
--   `textSans.xlarge()` -> 24px
--   `textSans.xxlarge()` -> 28px
--   `textSans.xxxlarge()` -> 34px
+<Story name="textSans">
```

Comment:
Wrapping each story in `<Canvas>` like: `<Canvas><Story> {....} </Story></Canvas>` might help to provide some nice visual delineation for each story by adding a box around it. Examples below to illustrate:

## Before

![Screenshot 2022-08-24 at 11 58 31](https://user-images.githubusercontent.com/1771189/186402201-d826914d-74cf-46a0-95e1-c51cc37f6c3e.png)


## After

![Screenshot 2022-08-24 at 11 58 19](https://user-images.githubusercontent.com/1771189/186402214-9ab7c8f6-4a0c-44c2-ae1f-ed6348f85118.png)

Reply:
Good shout.